### DISCUSSION TITLE
Crash after/during update (rel 20230522)

### DISCUSSION THREAD

**Discussion by jtritter** [rern/rAudio#1324] [Bug Reports]
Hi there,

raudio (the addons) told me about a new update (release 2020522). So I clicked on update... :-(
During the update process there was an error message  get files failed

And since then, nothing worked any more. When I go to the raspi via browser I only get a white screen, the source shows an end in the html header. Nothing works anymore.

Is there any chance to restart the update via command line (ssh access works, but no use of the web interface or the android app)? Or do I have reinstall the whole system?

Yours
Jürgen

**Comment by stymaster** [comment]
I had this issue too. Web interface completely broken, tried SSHing in to do "shutdown -r now", and nothing happened. Power-cycled, and problem persisted, I ended up re-installing.

**Comment by kan-ibal** [comment]
You can try manual update. Download:
`https://github.com/rern/rAudio/archive/refs/tags/20230522.zip` 
and copy all files from /srv directory via SSH, SFTP or directly to SDcard.

**Reply by jtritter** [reply]
Thank your for the idea. But unfortunately, that didn't work. The only thing I got was a JSON error :-(

**Comment by jtritter** [comment]
At least, I had to re-install the whole thing.
